pyclam/tests/utils.py

```diff
@@ -15,24 +15,3 @@ def linear_search(point: Data, radius: Radius, data: Data, metric: str):
         distances = cdist(point, batch, metric)[0]
         results.extend([p for p, d in zip(batch, distances) if d <= radius])
     return results
-
-
-def trace_lineage(cluster: Cluster, other: Cluster):  # TODO: Cover
```

Comment:
Is this still functional? Do we see a potential need for it in the future?

Reply:
It's not used anywhere and I don't see it being used anywhere in the future. If I'm wrong, it's still in the git history